bundles/org.openhab.binding.wundergroundupdatereceiver/README.md

```diff
@@ -0,0 +1,121 @@
+| winddir          | Number:Angle         | Current wind direction angle                                                                       |
+| windgustmph      | Number:Speed         | Current wind gust speed                                                                            |
+| windgustdir      | Number:Angle         | Wind gust direction angle                                                                          |
+| windspdmph_avg2m | Number:Speed         | 2 minute average wind speed                                                                        |
+| winddir_avg2m    | Number:Angle         | 2 minute average wind direction angle                                                              |
+| windgustmph_10m  | Number:Speed         | 10 minute average gust speed                                                                       |
+| windgustdir_10m  | Number:Angle         | 10 minute average gust direction angle                                                             |
+| tempf            | Number:Temperature   | Current outdoor temperature                                                                        |
+| indoortempf      | Number:Temperature   | Current indoor temperature                                                                         |
+| soiltempf        | Number:Temperature   | Current soil temperature                                                                           |
+| humidity         | Number:Dimensionless | Current humidity in percent                                                                        |
+| indoorhumidity   | Number:Dimensionless | Current indoor humidity                                                                            |
+| dewptf           | Number:Temperature   | Dew point                                                                                          |
+| soilmoisture     | Number:Dimensionless | Soil moisture in percent                                                                           |
+| leafwetness      | Number:Dimensionless | Leaf wetness in percent                                                                            |
+| rainin           | Number:Length        | Rain over the past hour                                                                            |
+| dailyrainin      | Number:Length        | Rain since the start of the day                                                                    |
+| weeklyrainin     | Number:Length        | Rain since the start of the week                                                                   |
+| monthlyrainin    | Number:Length        | Rain since the start of the month                                                                  |
+| yearlyrainin     | Number:Length        | Rain since the start of the year                                                                   |
```

Comment:
Dont include the units in the channel ID. UOM means it will display in mm in some countries.

Reply:
The channel ids match the request parameters. The goal of the binding is to collect the measurements the weather station would submit to wunderground.com.

Reply:
It would also be useful to be able to manually add synthetic channels to calculate various derivative measurements in rules triggered by weather station requests. This is where items like the heatindex could be calculated.

Reply:
Please see here https://www.openhab.org/docs/developer/guidelines.html#java-coding-style
The channel Ids must be in lowerCamelCase. It appears that you have now removed the units from the channel IDs from looking at the channels.xml file, however 2 things.

1. What is in the XML does not match the readme
2. The xml still uses dashes and not the naming convention in the link above.

Reply:
The channel ids map the request parameters. They are not supposed to be interpreted as conveying anything other than values submitted in the request. The descriptions are from the wunderground.com API documentation, so to match they have to match.